railgun/test/integration/suite_test.go

```diff
@@ -129,6 +137,29 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 		}
 		kubeconfig.Close()
 
+		// cleanup any previously existing CRDs
+		// TODO: can disable when we fix:
+		//       https://github.com/Kong/kubernetes-testing-framework/issues/19
+		crdCleanup := []string{
+			"kongclusterplugins.configuration.konghq.com",
+			"kongconsumers.configuration.konghq.com",
+			"kongingresses.configuration.konghq.com",
+			"kongplugins.configuration.konghq.com",
+			"tcpingresses.configuration.konghq.com",
+		}
+		for _, crd := range crdCleanup {
+			cmd := exec.CommandContext(ctx, "kubectl", "--kubeconfig", kubeconfig.Name(), "delete", "crd", crd)
+			stdout, stderr := new(bytes.Buffer), new(bytes.Buffer)
+			cmd.Stdout = stdout
+			cmd.Stderr = stderr
+			if err := cmd.Run(); err != nil {
+				if !strings.Contains(stderr.String(), "not found") {
+					fmt.Fprintln(os.Stdout, stdout.String())
+					panic(fmt.Errorf("%s: %w", stderr.String(), err))
+				}
+			}
+		}
```

Comment:
Is there any need for this beyond the current KTF behavior? If not, IMO this workaround is de facto `--skip-crds` with no toggle since it will always remove the chart CRDs, so we as well just [disable CRD creation there for now](https://github.com/Kong/kubernetes-testing-framework/issues/19#issuecomment-824315899) rather than working around it here.

Reply:
Sounds good :+1:

Reply:
https://github.com/Kong/kubernetes-testing-framework/pull/20 is created, I will block this on that and updating to a new release of KTF :+1: